server/webapp/WEB-INF/rails.new/app/assets/javascripts/lib/mithril-0.2.5.js

```diff
@@ -1399,10 +1399,11 @@
 			roots[index] = root
 			var currentComponent
 
+			var dummy = {controller: noop}
 			if (component) {
 				currentComponent = topComponent = component
 			} else {
-				currentComponent = topComponent = component = {controller: noop}
+        currentComponent = topComponent = component = dummy

```

Comment:
Yes — see https://github.com/lhorie/mithril.js/issues/727#issuecomment-244146781

@iamsumanth — could you move this into a new PR with clear links to why this fix was needed